src/LoadBalance/LB_Refine.cpp

```diff
@@ -91,11 +92,23 @@ void LB_Refine( const int FaLv )
    int    CFB_NSibEachRank[MPI_NRank];
    real  *CFB_BField=NULL;
 
+   bool SwitchFinerLevelsToWaveScheme = false;
+
    LB_Refine_GetNewRealPatchList( FaLv, NNew_Home, NewPID_Home, NNew_Away, NewCr1D_Away, NewCr1D_Away_IdxTable, NewCData_Away,
                                   NDel_Home, DelPID_Home, NDel_Away, DelCr1D_Away,
                                   RefineF2S_Send_NPatchTotal, RefineF2S_Send_PIDList,
-                                  CFB_SibLBIdx_Home, CFB_SibLBIdx_Away );
+                                  CFB_SibLBIdx_Home, CFB_SibLBIdx_Away,
+                                  SwitchFinerLevelsToWaveScheme );
+
+# if ( MODEL == ELBDM && ELBDM_SCHEME == ELBDM_HYBRID )
+// sync information whether refined levels are switched to wave scheme
+   int Send = SwitchFinerLevelsToWaveScheme;
+   int Recv;
+
+   MPI_Allreduce(&Send, &Recv, 1, MPI_INT, MPI_LOR, MPI_COMM_WORLD);
```

Comment:
I'm not sure if it's safe to use `MPI_LOR` for `MPI_INT`. What do you think? Check other places using `MPI_LOR` too.